x-pack/plugins/security_solution/public/timelines/components/timeline/data_providers/helpers.test.tsx

```diff
@@ -1103,4 +1104,10 @@ describe('helpers', () => {
       });
     });
   });
+
+  describe('convertIsOneOfQueryStringArrayToDisplayValue', () => {
+    it('converts a "is one of" query to correct format for a string array', () => {
+      expect(convertIsOneOfQueryStringArrayToDisplayValue(['a', 2, 'c'])).toBe('( a OR 2 OR c )');
```

Comment:
Consider removing the extra whitespace after the opening `(`, and before the closing `)`:

**Before**
`( a OR 2 OR c )`

**After (suggested)**
`(a OR 2 OR c)`

Reply:
changed 👌